contrib/cirrus/runner.sh

```diff
@@ -50,6 +50,10 @@ function _run_validate() {
     # Confirm compile via prior task + cache
     bin/podman --version
     bin/podman-remote --version
+
+    # FIXME FIXME FIXME: 2020-11-10: remove once Fedora 33 has FindBin
+    perl -MFindBin -e 0 &>/dev/null || dnf -y install perl-FindBin
+
```

Comment:
I'm on the fence, risk-wise.  Do you think it makes sense to instead of the above,  make use of VM image suffix 'c5402398833246208', since https://github.com/containers/automation_images/pull/39 is finished?

Reply:
I think you know my answer to that: I like small reviewable PRs, not complicated multi-element ones. This one is unfortunately too big already, but every element is necessary. I can submit a new PR once this merges, to use the new VM images and remove the dnf-install.